### ISSUE TITLE
Hook after carbon fields update vie WP Rest-Api

### ISSUE THREAD

**Issue by BenBroide** [htmlburger/carbon-fields#406]
When carbon fields are updated in the back end editor screen we could hook after the fields saved with carbon_fields_post_meta_container_saved. 
How could we hook after carbon fields are updated via WP Rest-Api?
Thanks

**Comment by atanas-angelov-dev** [comment]
I don't believe there is a suitable action in this case - I'll mark this issue as a feature request.

In the meantime, you could use the `carbon_fields_should_save_field_value` filter (Field.php @ line 436) but keep in mind that it's a filter not an action and that it executes BEFORE the data is saved to the database (but the field will already hold the new value).
